menu/menu.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * NOTE: 
+ * 
+ * This file defines menu entries for your plugin, if you do not need these, 
+ * remove the ../menu folder as well as the menu reference in the root index file (plugin entry point)
+ * 
+ * You may define as many entries as you would like, here we enable the first when 
+ * focusing a Camunda Platform/7 BPMN diagram and the second on any BPMN diagram.
+ * 
+ * Accelerators are optional.
+ * 
+ */
+
+module.exports = function (electronApp, menuState) {
+  return [{
+    label: 'Send a message to the console',
+    accelerator: 'CommandOrControl+=',
+    enabled: () => menuState.bpmn && menuState.platform === 'platform', 
+    action: () => console.log('📓 A custom menu entry was triggered')
```

Comment:
This does not work for me in practice, am I missing something?

Reply:
Alternative: use example like here https://github.com/camunda/camunda-modeler-plugins/blob/master/menu-plugin-example/menu/menu.js

Reply:
This may actually be a good place to add a note, unless there's an issue, this 'doesn't work' for you because the menu is executed on the main thread and not the render thread. What that means is that those console logs will actually be caught in vscode.

Reply:
I see. Make sense. Let's add a note and maybe link to the plugin docs, which also explain this?

Reply:
Done 👍